### PR TITLE
change from bias to bias_eta

### PR DESCRIPTION
**Pull request by londumas**
change from `bias` to `bias_eta`. Address issue https://github.com/igmhub/picca/issues/382.

### CODE REVIEW

**Comment by ngbusca** [comment]
@londumas you should check with @andreufont , bu I think `bias` should go to `bias_delta` not `bias_eta`.

**Comment by andreufont** [comment]
No, I think @londumas is right. Picca was calling "bias" what we would usually call bias_eta, i.e., the response of delta_F to a velocity gradient. 

I believe bias_delta, the response to a large scale overdensity, is already called this way in picca?

**Comment by londumas** [comment]
@ngbusca, I agree word for word. @andreufont is that good for you?

File: py/picca/test/data/config_xcf.ini [hunk 1]
```diff
@@ -36,7 +36,7 @@ in tracer2 = SiIII(1207)
 
 [parameters]
 
-bias_QSO  = 109.8 1. None None free
+bias_eta_QSO  = 109.8 1. None None free
```
Comment (andreufont) [inline py/picca/test/data/config_xcf.ini]: why do you set bias_eta_QSO to such a large number? I guess this is a test that has no impact anywhere else, but I was a bit surprised to see it.

**Comment by londumas** [comment]
@ngbusca, can you also do a review?

**Comment by ngbusca** [comment]
@londumas you should go ahead, don't wait for me

**Comment by londumas** [comment]
Thanks @andreufont and @ngbusca.

**Pull request merged**